cmd/porter/credentials.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/deislabs/porter/pkg/printer"
+	"github.com/spf13/cobra"
+)
+
+func buildCredentialsCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "credentials",
+		Aliases: []string{"cred"},
+		Short:   "Credentials commands",
+	}
+
+	cmd.AddCommand(buildCredentialsAddCommand(p))
+	cmd.AddCommand(buildCredentialsEditCommand(p))
+	cmd.AddCommand(buildCredentialsGenerateCommand(p))
+	cmd.AddCommand(buildCredentialsListCommand(p))
+	cmd.AddCommand(buildCredentialsRemoveCommand(p))
+	cmd.AddCommand(buildCredentialsShowCommand(p))
+
+	return cmd
+}
+
+func buildCredentialsAddCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "add",
+		Short: "Add Credential",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			p.PrintVersion()
```

Comment:
I suggest setting `Hidden: true` on anything that is wired but not implemented, just so that it doesn't show up in the helptext. You can still call it, but it won't be here confusing anyone using canary.